### ISSUE TITLE
how to add a mse loss

### ISSUE THREAD

**Issue by andeyeluguo** [junfu1115/DANet#93]
now I want to add an mse loss, but i can not figure out how to add. Should I edit the parallel.py ？or just use it.... I have change much, but it does not work.
When I want to compute the loss in one  gpu, when it runs in loss.backward, it refers that the tensor is in different gpu, the error, appeared. Somebody says that  the pytorch version is too low, and it is a bug. I do not know why.
